int32 req_int
---
bool success
string message
